src/tools.ts

```diff
@@ -115,8 +134,15 @@ export default function createTools(config: Config): Array<Tool> {
             command       : './vendor/bin/codecept run',
             filesToCheck  : [ 'codeception.yml.dist', 'codeception.yml' ],
             toolType      : ToolType.CODE_CHECK,
-        }
-    ]
+        },
+    ] as Tool[];
+
+    if (bcTool !== null) {
```

Comment:
yuck yuck yuck - would prefer to merge two sets of `Tool[]`, but for now, this will have to do.

Reply:
You could do something like:

```ts
const tools = [
    ...[bcTool].filter()
    <other tools>
];
```

Spread new array containing the `bcTool with filter removing null values.

Reply:
Tried that, but `.filter()` TSC wasn't able to infer `Tool[]` from it, due to too many dynamic expressions, I guess.

Reply:
`...([bcTool].filter() as Tool[])` will do that.